.github/workflows/django.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     strategy:
       max-parallel: 4
       matrix:
-        python-version: [ '3.8', '3.9', '3.10' ]
+        python-version: ['3.10' ]
```

Comment:
can we have symmetric spacing please 🙃

Reply:
Will add pre-commit hooks that track yaml as well :+1: